source/includes/_resource.videos.md

```diff
@@ -548,3 +576,156 @@ vhx.videos.files({
   <p>If building a customer-based application, <code>Authorizations</code> must be used so our player can properly authenticate and record analytics for the viewing customer.</p>
   <p>The returned source href has a TTL / expiration, so it is advised that the files API call happens just prior to a stream action.</p>
 </section>
+
+<h3 class="text-2 head-4 text--navy text--bold is-api margin-top-large margin-bottom-medium"
+id="collections-update">Update a Video</h3>
+
+> <h5 class="head-5 text--white margin-bottom-medium">Update a Video</h5>
+
+> Definition
+
+```shell
+POST /videos/:id
+```
+
+```ruby
+Vhx::Video.update()
+```
+
+```node
+vhx.videos.update();
+```
+
+```javascript
+// Not available for client-side requests.
+```
+
+```php
+<?php\VHX\Videos::update();
+```
+
+> Example Request
+
+```shell
+$ curl -X POST "https://api.vhx.tv/videos/1" \
```

Comment:
Ah, copy and paste fail. I think other #update sections might be listing `POST` as the method as well. Will fix those, nice catch!